inmanta.spec

```diff
@@ -62,9 +62,15 @@ BuildRequires:  openssl-devel >= 1:1.1.1
 Requires:       openssl >= 1:1.1.1
 %endif
 
+%if "%{#undotted_python_version}" == "2"
 BuildRequires:  python%{undotted_python_version}-devel
 Requires:       python%{undotted_python_version}
 Requires:       python%{undotted_python_version}-devel
+%else
+BuildRequires:  python%{python_version}-devel
+Requires:       python%{python_version}
+Requires:       python%{python_version}-devel
```

Comment:
I'm curious, why do we have to use undotted_python_version for python versions <= 3.9 ?

Reply:
On Rocky Linux. The python package for Python3.9 is called `python39`. The package for Python3.11 is called `python3.11`. The same is true for the `-devel` packages.